docs/configuration.md

```diff
@@ -90,83 +90,97 @@ Settings like this are typically be provided by the language-specific LSP client
 ## Configuring your project build
 
 `haskell-language-server` has to compile your project in order to give you diagnostics, which means that it needs to know how to do so.
-This is handled by the [hie-bios](https://github.com/mpickering/hie-bios) project.
+This is handled under the hood by the [hie-bios](https://github.com/mpickering/hie-bios) application.
+In turn, `hie-bios` needs some configuration to identify all files, GHC options, etc., needed to compile a project.
 
-**For a full explanation of how `hie-bios` determines the project build configuration, and how to configure it manually, refer to the [hie-bios README](https://github.com/mpickering/hie-bios/blob/master/README.md).**
+There are several ways to provide this configuration to `hie-bios`, detailed below.
 
-At the moment, `haskell-language-server` has support to automatically detect your project build configuration to handle most use cases.
+### Implicit configuration
+If no `hie.yaml` file is present, `haskell-language-server` automatically detects your `hie-bios` configuration.
+**For most cases, this works just fine, and is the recommended way.**
 
-*So using a explicit `hie.yaml` file will not likely fix your ide setup*. It will do it almost only if you see an error like `Multi Cradle: No prefixes matched`
+See [the hie-bios documentation](https://github.com/haskell/hie-bios#implicit-configuration) for more information on implicit configuration.
```

Comment:
I'm not sure this is quite accurate. I believe we in fact will use `implicit-hie` in this instance, not the `hie-bios` implicit auto-configuration...

Reply:
Done in 739e58feeaaa80c1314686bd3f269986da2c5cdb